vulcan/lib/server/workflows/scripts/parse_record_selections.py

```diff
@@ -6,17 +6,18 @@
 pdat = input_json("project_data")[project_name]
 selection_atts = pdat['selection_options']
 
-selected = input_json('selected_options')
+# because a is the input name from the previous CWL input step
+selected = input_json('selected_options')["a"]
 
 magma = connect()
 
 # Create filters for all the 'select-bys' that the value for this attribute
 # must be among the options selected in the previous step.
 filters = []
-for set in list(selection_atts.keys()):
-    if len(selected[set]) > 0:
+for selected_model in list(selection_atts.keys()):
```

Comment:
`set` is a keyword in Python, so I think it's safer to use a different variable name here.

Reply:
ahhh right!  That's a base data-type isn't it!  Oops.  Can we change this to `target` instead of `selected_model`?

`selected_model` might be confusing. The direct target is the 'label' of the `<label>: <model>#<attribute>` definition, and it's foreseeable that a project definition might actually include two separate attributes of the same model in the future.